typescript/sdk/src/token/TokenStandard.ts

```diff
@@ -60,6 +62,8 @@ export const TOKEN_STANDARD_TO_PROTOCOL: Record<TokenStandard, ProtocolType> = {
   EvmHypSyntheticRebase: ProtocolType.Ethereum,
   EvmHypXERC20: ProtocolType.Ethereum,
   EvmHypXERC20Lockbox: ProtocolType.Ethereum,
+  EvmIntent: ProtocolType.Ethereum,
+  EvmIntentNative: ProtocolType.Ethereum,
```

Comment:
`EmvIntent` and `EvmIntentNative` are the standars that will identify the intents